medicines/doc-index-updater/Makefile

```diff
@@ -6,6 +6,14 @@ tag := latest
 default: ## run locally
 	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo run
 
+.PHONY: test
+test: ## run locally
+	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo test $$TEST
```

Comment:
Can we document this behaviour somewhere? Add a "To run tests" section in the readme?

Reply:
I would suggest that the make file is part of the documentation isn’t it?

Alternatively

We could make a ‘make help’ ?

Reply:
ah `make help` does this already - I wondered what #run locally was all about

Reply:
@craiga do `make help` to see this in action with the latest of this branch

Reply:
Did the readme entry too